Nos/Views/SideMenuContent.swift

```diff
@@ -109,6 +109,9 @@ struct SideMenuContent: View {
                     AboutView()
                 }
             }
+            .navigationDestination(for: EditProfileDestination.self) { destination in
```

Comment:
What case does this cover? I remember testing your previous PR that didn't have this and the EditProfile still worked. I also tested removing this and still works. So I wonder if covers a case I didn't consider or it it was just in case to leave the code as it were before the refactor.

Reply:
I needed this to cover the "Complete my profile" button that shows up in the sidebar after creating a fresh account. I just tested removing this navigationDestination and confirmed that it's needed.
![Screenshot 2024-06-10 at 9 54 53 AM](https://github.com/planetary-social/nos/assets/1165004/abdae8fc-8b9d-49b8-b0fa-890b8ad9607d)